src/MuTalk-Model/MutationTestingBudget.class.st

```diff
@@ -8,8 +8,16 @@ Class {
 	#package : 'MuTalk-Model'
 }
 
+{ #category : 'instance creation' }
+MutationTestingBudget class >> for: aConstraint [
+
+	^ self new
+		  constraint: aConstraint;
+		  yourself
+]
+
 { #category : 'validation' }
-MutationTestingBudget >> check: aCollection [
+MutationTestingBudget >> check: partialCollection with: initialCollection [
```

Comment:
Maybe we should rename this method in a subsequent PR.

`check:with:` does not give enough information about what it does.
We should use a more intention revealing selector.